views/partials/breach-groups.hbs

```diff
@@ -1,5 +1,5 @@
         {{#each firstFourBreaches }}
-          {{> breach-card cardType=./../cardType}}
+          {{> breach-card cardType=./../cardType LOGOS_ORIGIN=./../LOGOS_ORIGIN}}
```

Comment:
This doesn't seem to be the right path to the parent context with `LOGOS_ORIGIN` in it. :/ The dashboard breach groups still render without the logos origin value.

Reply:
Might be easier to do this in the `makeBreachCards` helper.  Otherwise, we'll have to send the value like this everywhere there are breach cards or breach images (latest breach, scan results, breach details, dashboard, emails) which could be painful.

Reply:
Might be easier to do this in the `makeBreachCards` helper.

Reply:
I'm not an .hbs expert (but I should really be better at it by this point), but is there a way to just specify globals in Handlebars instead of trying to pass relative pointers to variables?

Reply:
I looked for a way to access "globals" in the partials, but couldn't find one. But it sounds like @lesleyjanenorton 's idea to move this logic into the helper function will effectively avoid the relative pointers too.

Reply:
FWIW, it looks like we might be able to use the `helpers` to inject some globals: https://github.com/pdehaan/express-hbs-helpers-test
Not sure if its a good or bad idea, or if it works outside of my silly little sandbox test case, but 🤷‍♂.

<kbd>/giphy "the more you know"</kbd>